src/Nethermind/Nethermind.Core.Test/Container/FunctionalGenesisPostProcessor.cs

```diff
@@ -3,14 +3,13 @@
 
 using System;
 using Nethermind.Blockchain;
-using Nethermind.Evm.State;
 
 namespace Nethermind.Core.Test.Container;
 
-public class FunctionalGenesisPostProcessor(IWorldState state, Action<Block, IWorldState> postProcessor) : IGenesisPostProcessor
+public class FunctionalGenesisPostProcessor(Action<Block> postProcessor) : IGenesisPostProcessor
 {
     public void PostProcess(Block genesis)
     {
-        postProcessor.Invoke(genesis, state);
+        postProcessor.Invoke(genesis);
```

Comment:
I always wonder why you use `Invoke` rather than: 

```suggestion
        postProcessor(genesis);
```